source/help/submit_tex.md

```diff
@@ -21,17 +21,17 @@
     - Our plan is for arXiv's "current" version to closely follow the annual TeX Live releases. In the past, we often went several years between TeX updates.
- 1. The AutoTeX system we are updating would attempt to determine which files in a submission were part of the main document and which were not. We are no longer going to do this.
+ 1. Submission 1.0 would attempt to determine which files in a submission were part of the main document and which were not. We are no longer going to do this.
       - If there were multiple files ending in `.tex`, it would create PDFs for those extra files and append them to main paper's PDF. Now, if authors want a .tex file to be part of their main paper, they should use \include or \input commands to include the file. See [https://www.baeldung.com/cs/latex-include-vs-input](https://www.baeldung.com/cs/latex-include-vs-input).
       - If authors need the previous apppend behavior for a submission with multiple `.tex` files, please follow the instructions here: [https://info.arxiv.org/help/00README.html](https://info.arxiv.org/help/00README.html).
       - If image files (JPG, PNG, PDF) were found, they would be rendered and appended verbatim to the main paper's PDF. If authors want to include images anywhere in their paper, they should use the normal TeX contructs for this. See [https://latex-tutorial.com/tutorials/figures/](https://latex-tutorial.com/tutorials/figures/). 
   
-  1. The AutoTeX system would preload the LaTeX hyperref package to LaTeX submissions that did not already include it. This package adds various references to active links in the PDF document (e.g., clicking on a reference jumps to the entry in the bibliography section). We will no longer automatically try to add this package. (Note – before you add a \usepackage{hyperref} to your main TeX file, check to see if the template you are using already has a reference by checking the PDF for clickable links; most do).
- 1. The AutoTeX system would look for any references in a paper that looked like an arXiv paper ID (such as arXiv:2402.08954, or 2402.08954) and turn the ID into a hyperlink to the paper on arXiv. We will not do that anymore. Authors should just write out `\href{https://doi.org/10.48550/arXiv.2402.08954}{2402.08954}`. See the [Hyperref documentation for more info](https://mirror.math.princeton.edu/pub/CTAN/macros/latex/contrib/hyperref/doc/hyperref-doc.html).
+  1. Submission 1.0 would preload the LaTeX hyperref package to LaTeX submissions that did not already include it. This package adds various references to active links in the PDF document (e.g., clicking on a reference jumps to the entry in the bibliography section). We will no longer automatically try to add this package. (Note – before you add a \usepackage{hyperref} to your main TeX file, check to see if the template you are using already has a reference by checking the PDF for clickable links; most do).
```

Comment:
Suggestion: Change the dash after "(Note -" to a colon "(Note:" to match other usage on this page.